src/lib/components/SelectList.svelte

```diff
@@ -0,0 +1,73 @@
+<!--
+@component
+A component to display menu options in a list.
+-->
+<script lang="ts">
+    import { createEventDispatcher } from 'svelte';
+    import config from '$lib/data/config';
+    export let options = [''];
+
+    const dispatch = createEventDispatcher();
+    $: rows = Math.ceil(options.length);
+    const colors = (type: string, key: string) =>
+        config.themes.find((x) => x.name === 'Normal').colorSets.find((x) => x.type === type)
+            ?.colors[key];
+    const textColor = colors('main', 'ChapterButtonTextColor');
+    let tableColor = colors('main', 'BackgroundColor');
```

Comment:
Use the $s store to retrieve colors. I believe everything you need is in under the ui.select-list-button key.